dev/breeze/src/airflow_breeze/commands/testing.py

```diff
@@ -15,23 +15,32 @@
 # specific language governing permissions and limitations
 # under the License.
 
+import os
 import sys
 from typing import Tuple
 
 import click
 
 from airflow_breeze.build_image.prod.build_prod_params import BuildProdParams
 from airflow_breeze.commands.common_options import (
+    option_db_reset,
     option_dry_run,
     option_github_repository,
     option_image_name,
     option_image_tag,
+    option_integration,
     option_python,
     option_verbose,
 )
+from airflow_breeze.commands.custom_param_types import BetterChoice
 from airflow_breeze.commands.main import main
+from airflow_breeze.global_constants import ALLOWED_TEST_TYPES
+from airflow_breeze.shell.enter_shell import check_docker_is_running, check_docker_resources
+from airflow_breeze.shell.shell_params import ShellParams
 from airflow_breeze.utils.console import get_console
+from airflow_breeze.utils.docker_command_utils import construct_env_variables_docker_compose_command
 from airflow_breeze.utils.run_tests import run_docker_compose_tests
+from airflow_breeze.utils.run_utils import run_command
 
 TESTING_COMMANDS = {
     "name": "Testing",
```

Comment:
You need to add "breeze test" command  here so that it is displayed in correct section in `breeze --help`, also "test-specific" options should have it's own section (TESTING_PARAMETERS below). This makes the command --help output better organized.